.github/workflows/build.yml

```diff
@@ -63,59 +63,49 @@ jobs:
           version: ${{ matrix.qt-version }}
 
       # WINDOWS
-
       - name: Cache conan packages
         if: startsWith(matrix.os, 'windows')
         uses: actions/cache@v2.1.4
         with:
           key: ${{ runner.os }}-conan-${{ hashFiles('**/conanfile.txt') }}-20210307
           path: C:/.conan/
 
-      - name: Install dependencies (Windows)
+      - name: Download conan
         if: startsWith(matrix.os, 'windows')
         run: |
-            choco install conan -y
-
-            refreshenv
-        shell: cmd
+            Invoke-WebRequest -Uri "https://github.com/AnotherFoxGuy/conan-onefile/releases/latest/download/conan.exe" -OutFile "C:\vcpkg\conan.exe"
```

Comment:
Let's revert these changes and keep install conan through choco if possible

Reply:
The reason I changed it is because installing conan using choco takes anywhere between 3 to 8 minutes, and this method only takes 5 seconds
Also, this removes the need for a second "Add conan to path" action

Reply:
Yeah let's leave that for another time though - the windows CI is still not the one we have to wait for in the github actions

Reply:
maybe we could just cache the choco results instead?

Reply:
either way something for another PR